docs/new-architecture-app-modules-android.md

```diff
@@ -189,10 +240,13 @@ Then, you can provide the class you created to your `ReactNativeHost`. You can l
 
 Once you located it, you need to add the `getReactPackageTurboModuleManagerDelegateBuilder` method as from the snippet below:
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="java">
+
 ```java
 public class MyApplication extends Application implements ReactApplication {
 
-    private final ReactNativeHost mReactNativeHost =
+    private final ReactNativeHost reactNativeHost =
```

Comment:
Sorry I was not precise before. Let's use `mReactNativeHost` for Java and `reactNativeHost` for Kotlin.

That convention was in place for Java only:
https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions

Reply:
Fixed